src/passes/rejectUnsupportedFeatures.ts

```diff
@@ -140,6 +140,15 @@ export class RejectUnsupportedFeatures extends ASTMapper {
       if (!(node.parent instanceof MemberAccess && node.parent.memberName === 'sender')) {
         this.addUnsupported(`msg object not supported outside of 'msg.sender'`, node);
       }
+    } else if (node.name === 'block' && node.vIdentifierType === ExternalReferenceType.Builtin) {
+      if (
+        node.parent instanceof MemberAccess &&
+        ['coinbase', 'chainid', 'gaslimit', 'basefee', 'difficulty'].includes(
+          (<MemberAccess>node.parent).memberName,
```

Comment:
I think this type hint is not necessary due to the use of `instanceof` previously